src/components/periodic-report/periodic-report.repository.ts

```diff
@@ -111,17 +111,72 @@ export class PeriodicReportRepository extends DtoRepository(IPeriodicReport) {
       );
   }
 
+  async currentProjectReport(
+    projectId: string,
+    reportType: ReportType,
+    startDate: DateTime
+  ) {
+    return await this.db
+      .query()
+      .match([
+        node('project', 'Project', { id: projectId }),
```

Comment:
```suggestion
        node('baseNode', 'BaseNode', { id: projectId }),
```
Now it works for engagements as well.